lambdas/handlers/document_manifest_by_nhs_number_handler.py

```diff
@@ -33,68 +29,34 @@
 @override_error_check
 def lambda_handler(event, context):
     request_context.app_interaction = LoggingAppInteraction.DOWNLOAD_RECORD.value
-
-    logger.info("Starting document manifest process")
-
     try:
         nhs_number = event["queryStringParameters"]["patientId"]
         doc_type = extract_document_type(event["queryStringParameters"]["docType"])
         request_context.patient_nhs_no = nhs_number
 
-        zip_output_bucket = os.environ["ZIPPED_STORE_BUCKET_NAME"]
-        zip_trace_table_name = os.environ["ZIPPED_STORE_DYNAMODB_NAME"]
-        # zip_trace_ttl = os.environ["DOCUMENT_ZIP_TRACE_TTL_IN_DAYS"]
-
-        dynamo_service = DocumentService()
-        documents = dynamo_service.fetch_available_document_references_by_type(
-            nhs_number, doc_type
-        )
-
-        if not documents:
-            return ApiGatewayResponse(
-                204, "No documents found for given NHS number and document type", "GET"
-            ).create_api_gateway_response()
-
         logger.info("Starting document manifest process")
-        document_manifest_service = DocumentManifestService(
-            nhs_number=nhs_number,
-            documents=documents,
-            zip_output_bucket=zip_output_bucket,
-            zip_trace_table=zip_trace_table_name,
+        document_manifest_service = DocumentManifestService(nhs_number)
```

Comment:
Not sre NHS number should be a part of the object init, I don't see the value added by having this as a class level parameter vs adding it in as a param to the method call since it's only used once to get the docs list from Dynamo

Reply:
It's used to generate the zip file name upon initialisation, which is used across all class methods so I think it seems sensible to keep in the init